pkg/storage/memory/memory.go

```diff
@@ -18,14 +18,37 @@ type (
 
 	storageMem struct {
 		sync.RWMutex
-		store map[string]memStorageSecret
+		storePruneTimer time.Ticker
+		store           map[string]memStorageSecret
 	}
 )
 
 // New creates a new In-Mem storage
 func New() storage.Storage {
-	return &storageMem{
-		store: make(map[string]memStorageSecret),
+	store := &storageMem{
+		storePruneTimer: *time.NewTicker(time.Minute),
```

Comment:
Is there any reason to dereference the pointer instead of just holding it? 🤔

Reply:
No good reason, now that you've pointed it out it doesn't make sense. Corrected.